src/Microsoft.PowerShell.ScheduledJob/ScheduledJobDefinition.cs

```diff
@@ -2300,7 +2300,7 @@ internal string FQEID
     /// <summary>
     /// Simple string formatting helper.
     /// </summary>
-    internal class StringUtil
+    internal static class StringUtil
```

Comment:
ScheduledJob module is not being built in the code base today, so please undo the change here.
Actually, we may need to think about remove the code that is not used today.

Reply:
:-) LocalAccounts/StringUtil.cs too.

Reply:
@daxian-dbw I guess work to remove uncompiled code is covered by #9960?